drivers/sensor/lsm6ds0/lsm6ds0.h

```diff
@@ -314,7 +314,7 @@
 
 #define SENSOR_G_DOUBLE		(SENSOR_G / 1000000.0)
 #define SENSOR_PI_DOUBLE	(SENSOR_PI / 1000000.0)
-#define SENSOR_DEG2RAD_DOUBLE	(180.0 / SENSOR_PI_DOUBLE)
+#define SENSOR_DEG2RAD_DOUBLE	(SENSOR_PI_DOUBLE / 180)
```

Comment:
Is there a reason you're not using `sensor_degrees_to_rad` defined in `include/drivers/sensor.h`?

Reply:
> sensor_degrees_to_rad
This driver is very old (2016). I'm not sure but maybe the sensor_degrees_to_rad() was not available at that time. Since the LSM6DS0 component is old and now obsolete, can we keep it as it is? For the new components (either the ones that are currently using SENSOR_DEG2RAD_DOUBLE) I may do a separate commit.

Currently I see the following:

```iis2iclx/iis2iclx.h:#define SENSOR_DEG2RAD_DOUBLE			(SENSOR_PI_DOUBLE / 180)
ism330dhcx/ism330dhcx.h:#define SENSOR_DEG2RAD_DOUBLE			(SENSOR_PI_DOUBLE / 180)
lsm6ds0/lsm6ds0.h:#define SENSOR_DEG2RAD_DOUBLE	(SENSOR_PI_DOUBLE / 180)
lsm6dsl/lsm6dsl.h:#define SENSOR_DEG2RAD_DOUBLE			(SENSOR_PI_DOUBLE / 180)
lsm6dso/lsm6dso.h:#define SENSOR_DEG2RAD_DOUBLE			(SENSOR_PI_DOUBLE / 180)
```
Only lsm6ds0 is obsolete. I can do a separate PR for all of them (even lsm6ds0)

Reply:
Sounds good, thanks.